### PR TITLE
fix(pyroscope): include popular k8s labels in pyroscope.ebpf k8s example docs

### PR DESCRIPTION
**Pull request by korniltsev**
#### PR Checklist

- [ ] CHANGELOG.md updated
- [ ] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: docs/sources/flow/reference/components/pyroscope.ebpf.md [hunk 1]
```diff
@@ -192,18 +192,47 @@ discovery.kubernetes "all_pods" {
...
+    regex = "Succeeded|Failed"
+    source_labels = ["__meta_kubernetes_pod_phase"]
+  }
   rule {
     action = "replace"
-    replacement = "${1}/${2}"
-    separator = "/"
+    regex = "(.*)@(.*)"
+    replacement = "ebpf/${1}/${2}"
+    separator = "@"
```
Comment (simonswine) [inline docs/sources/flow/reference/components/pyroscope.ebpf.md:208]: Not quite sure why you switched to `@`, but I guess that will not surface anywhere else

**Review by simonswine — APPROVED** [review]
LGTM

File: docs/sources/flow/reference/components/pyroscope.ebpf.md (hunk 1 above)
Comment (korniltsev) [reply docs/sources/flow/reference/components/pyroscope.ebpf.md:208]: I just copy pasted it from our jsonnet, and it was probably copypasted to the jsonnet from somewhere else

Comment (korniltsev) [reply docs/sources/flow/reference/components/pyroscope.ebpf.md:208]: Screenshot 2023-09-06 at 8 17 19 AM

https://grafana.com/blog/2022/03/21/how-relabeling-in-prometheus-works/

**Pull request merged**